spec/typed_parameters/comparison_spec.rb

```diff
@@ -25,10 +25,25 @@
       end
 
       context "when the request body does not adhere to the rules format" do
-        let(:name) { 1_000 }
+        context "when a non-specified parameter is included in the request body" do
+          let(:params) { { name: "M@", email: "matt@zipmark.com" } }
```

Comment:
@kurko do you think 'extra' params should simply be ignored, or should we actually mark the request as being invalid (this is what we're doing now).

Example:

```ruby
Rules = { data: Hash }

# the `foo` param is not whitelisted
param = { data: {}, foo: "bar" }
```